docs/velocity/admin/getting-started/README.mdx

```diff
@@ -112,7 +112,7 @@ minigames = "127.0.0.1:30068"
 
 # In what order we should try servers when a player logs in or is kicked from a server.
 try = [
-  "lobby"
+  "lobby", "factions"
```

Comment:
Feels weird to have these on the same line when the list is expanded?

Reply:
okay